app/helpers/locations_helper.rb

```diff
@@ -25,18 +25,18 @@ def selection_options(resource)
 
   def display_location_attributes(location)
     base_attributes = [
-      { name: "Postcode", value: location.postcode },
-      { name: "Local authority", value: location.location_admin_district },
-      { name: "Location name", value: location.name, edit: true },
-      { name: "Total number of units at this location", value: location.units },
-      { name: "Common type of unit", value: location.type_of_unit },
-      { name: "Mobility type", value: location.mobility_type },
-      { name: "Code", value: location.location_code },
-      { name: "Availability", value: location_availability(location) },
+      { name: "Postcode", value: location.postcode, attribute: "postcode" },
+      { name: "Location name", value: location.name, attribute: "name" },
```

Comment:
Name is above Local authority in the designs, even though they're the other order in the flow. This seems a bit odd to me but I can see why you'd want postcode and name at the top as they are more identifying (and show up in the details page as well as CYA)